ote/src/cljs/ote/views/monitor.cljs

```diff
@@ -55,29 +66,70 @@
+                                                                     "rgb(255,102,153)"
+                                                                     "rgb(235,102,204)"]
+                                                   :label " Palvelut tyypeittäin "}]}
+          monthly-types (:monthly-types monitor-data)
+          tertiili-types (:tertiili-types monitor-data)]
       (fn []
-        [:div {:style {:width "50%"}}
-         [:h2 "Valvontanäkymä"]
-         [:p "Valvontanäkymässä voit tarkastellla liikkumispalveluiden tuottamiseen osallistuvien yritysten kokonaismäärää, sekä määriä jaoteltuina liikkumispalvelutyyppien mukaan. Voit myös ladata kaikkien esitettyjen kuvaajien tiedot, sekä liikkumispalveluiden tuottamiseen osallistuvien yritysten tiedot (Y-tunnus, toiminimi sekä tuotetut liikkumispalvelutyypit) CSV-tiedostona."]
-         [:p "Liikkumispalveluiden tuottamiseen osallistuvat yritykset"]
-         [ui-common/linkify "/admin/reports/monitor/csv/monthly-companies" "Lataa kuvaajan tiedot CSV:nä"]
-         [barchart-inner "chart-companies-by-month" companies-by-month-data]
-         [:p "Palveluntuottajien tämänhetkinen lukumäärä liikkumispalvelutyypeittäin"]
-         [ui-common/linkify "/admin/reports/monitor/csv/company-service-types" "Lataa kuvaajan tiedot CSV:nä"]
-         [doughnut-inner "chart-share-by-type" provider-share-by-type-data]
-         [:p "Tuottajien lukumäärä jaoteltuna liikkumispalvelutyypin mukaan"]
-         [ui-common/linkify "/admin/reports/monitor/csv/monthly-companies-by-service-type" "Lataa kuvaajan tiedot CSV:nä"]
-         [barchart-inner "chart-type-by-month" (:monthly-types monitor-data)]]))
+        [:div
+         [:h1 " Valvontanäkymä "]
+         [:p "Valvontanäkymässä voit tarkastella liikkumispalveluiden tarjoajien kokonaismäärän
+              kehitystä kokonaisuutena, sekä jaoteltuna liikkumispalvelutyyppien mukaan."]
+         [:p "Termi "
+          [:strong "Liikkumispalveluiden"] " tarjoajat tarkoittaa sekä NAPiin rekisteröityneitä "
+          [:strong " liikkumispalveluiden tuottajia"] ", että sellaisia yrityksiä, joista on maininta NAP:ssa
+          julkaistuiden palveluiden tiedoissa (ns. " [:strong "palveluiden tuottamiseen osallistuvat yritykset"] ").
+          Esimerkki palvelun tuottamiseen osallistuvasta yrityksestä on yksityinen elinkeinon harjoittaja,
+          joka ajaa ainoastaan taksivälityskeskuksen kyytejä."
+          ]
+         [:h2 " Liikkumispalveluiden tarjoajien lukumäärän kehitys kuukausittain "]
+         [ui-common/linkify " /admin/reports/monitor/csv/monthly-companies " " Lataa kuvaajan tiedot CSV:nä "]
+         [:div {:style {:width " 75 % "}}
+          [barchart-inner " bar-companies-by-month " nil companies-by-month-data]]
+
+         [:h2 " Liikkumispalveluiden tarjoajien lukumäärän kehitys tertiileittäin "]
+         [ui-common/linkify " /admin/reports/monitor/csv/tertiili-companies " " Lataa kuvaajan tiedot CSV:nä "]
```

Comment:
Just double checking that is linkify without url base ok?

Reply:
yes, I think so